test/test_sparse.py

```diff
@@ -1080,6 +1080,46 @@ def test_shape(di, dj, dk, nnz):
         test_shape(1000, 0, 100, 0)
         test_shape(1000, 100, 0, 0)
 
+    @cpu_only
```

Comment:
should you also test on cuda?

Reply:
This is because sspaddmm is not implemented on CUDA